libmamba/src/core/util_os.cpp

```diff
@@ -89,6 +90,29 @@ namespace mamba
 #endif
     }
 
+    fs::u8path get_libmamba_path()
+    {
+        fs::u8path libmamba_path;
+#if defined(__linux__) || __APPLE__ || __MACH__
+        Dl_info dl_info;
+        if (dladdr(reinterpret_cast<void*>(get_self_exe_path), &dl_info))
```

Comment:
To avoid dependencies to other functions changes, maybe trying to look for this function `get_libmamba_path` instead of another would work well?

Reply:
That's a mistake from me and I meant to use `get_libmamba_path` in place of `get_self_exe_path`.